
<launch>
	<!-- load the Parameters configuration file -->
	<!-- yaml files for navigation and object locations -->
	

	<node pkg="follow_me"
	      type="check_elevator_lila.py"
	      name="CheckElevator_node">
	
	</node>
	<node pkg="navigation_states"
	      type="reverse_node.py"
	      name="NavigationBack_node">
	</node>
	
	<node pkg="follow_me"
	      type="look_at_id.py"
	      name="move_head_look_id">
	</node>
 

</launch>
